doc/index.txt

```diff
@@ -168,6 +168,48 @@ Community
 
 * Come visit us in Montreal! Most developers are students in the LISA_ group at the `University of Montreal`_.
 
+.. _support:
+
+Help!
+=====
+
+How to get help
+---------------
+
+The appropriate venue for seeking help depends on the kind of question you have.
+
+* How do I? -- `theano-users`_ mailing list or `StackOverflow`_
+* I got this error, why? -- `theano-users`_ mailing list or `StackOverflow`_
```

Comment:
This section and the next one is a great idea. My only comment, can you add in that line that they should give the full error message, even if it is big? This help prevent ping/pong email to query the needed information.

Also, can you rebase on top of master to remove the many merge commit? This will make the history cleaner. If you aren't familiar with rebase, I can guide you or I can do it.

thanks